modules/encrypt/classes/Kohana/Encrypt/Engine/Openssl.php

```diff
@@ -10,83 +9,50 @@
  *
  * The Cipher
  * :  A [cipher](http://php.net/manual/en/openssl.ciphers.php) determines how the encryption
- *    is mathematically calculated. By default, the "AES-256-CBC" cipher
- *    is used.
+ *    is mathematically calculated. By default, the "AES-256-CBC" cipher is used.
+ *
  * @package    Kohana/Encrypt
  * @category   Security
- * @author     Kohana Team
- * @copyright  (c) Kohana Team
+ * @author     Koseven Team
+ * @copyright  (c) 2007-2012 Kohana Team
+ * @copyright  (c) 2016-2018 Koseven Team
  * @license    https://koseven.ga/LICENSE.md
  */
 class Kohana_Encrypt_Engine_Openssl extends Kohana_Encrypt_Engine
 {
-    use Traits_Encrypt_Iv;
```

Comment:
Code looks fine, this method can be overwritten in mcrypt, that's not a problem. I just wanna make sure you're not one of those people that never use traits, because someone told you they're bad (I have this kind of people at my work) 😄

Reply:
Hahaha no basicly i like them and also use them a lot. But since this is a common function it can be added to the parent class and overwritten if needed.